docs/runbook.md

```
@@ -396,7 +399,7 @@ Two tiers, run two different ways — see [tests/README.md](../tests/README.md)
 for the full rationale. In short:
 
 ```console
-$ uv run pytest -v                                                    # 190 passed
+$ uv run pytest -v                                                    # 201 passed
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)